### PR TITLE
Mp profile views

### PR DESCRIPTION
**Pull request by pattmax00**
Fixed the crash when viewing groups page, other than that it is identical to my previous pull request (closed).

### CODE REVIEW

**Review by Wil-LaLonde — COMMENTED** [review]
Check comments, thanks!

File: AcademicReward/AcademicReward.csproj [hunk 1]
```diff
@@ -98,4 +98,4 @@
 	  </MauiXaml>
 	</ItemGroup>
 
-</Project>
```
[the remark this answers is not in the document]
Comment (pattmax00) [reply AcademicReward/AcademicReward.csproj:101]: Visual Studio has just been adding them for me automatically, I attached the local version of mine from this branch.
[csproj.zip](https://github.com/Wil-LaLonde/CS341-Software-Project/files/9956256/csproj.zip)
Not sure why they don't seem to show up in git. On my local branch it says the only change is the page I'm working on now, and it has all my other pages listed (all my changes other than the CreateGroup view are pushed.
Capture

File: AcademicReward/Views/EditProfilePage.xaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="AcademicReward.Views.EditProfilePage"
+             Title="Edit Profile">
+
+    <StackLayout VerticalOptions="Center" HorizontalOptions="Center">
+        <Entry Placeholder="New Username" />
+        <Entry Placeholder="New Password" />
```
[the remark this answers is not in the document]
Comment (pattmax00) [reply AcademicReward/Views/EditProfilePage.xaml:9]: yes 👍🏻

File: AcademicReward/AcademicReward.csproj (hunk 1 above)
[the remark this answers is not in the document]
Comment (pattmax00) [reply AcademicReward/AcademicReward.csproj:101]: Now that I'm looking closer it doesn't look like I had to add any entries here for the History and Edit page. Very odd.

[the remark this answers is not in the document]
Comment (Wil-LaLonde) [reply AcademicReward/AcademicReward.csproj:101]: As long as things are working, I'm fine with them not being there. If something breaks, at least we know where to look.

**Pull request closed without merging**